databases/pgpointcloud/Portfile

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github  1.0
+
+github.setup        pgpointcloud pointcloud 1.2.5 v
+revision            0
+github.tarball_from archive
+
+name                pgpointcloud
+license             BSD-3-Clause
+categories          databases gis
+maintainers         {yahoo.com:n_larsson @nilason} openmaintainer
+
+description         pgPointcloud is a PostgreSQL extension for storing point cloud (LIDAR) data
+long_description    {*}${description}
+
+homepage            https://pgpointcloud.github.io/pointcloud/
+
+checksums           rmd160  54e0784994a458e9591ea416caf7ae106c0d7007 \
+                    sha256  f3924f283345f2da46a971e65f0c6ce602640ecf88a2057293443ec2a7a56774 \
+                    size    348678
+
+configure.cmd       autoupdate && ./autogen.sh && ./configure
+
+depends_build       port:autoconf \
```

Comment:
Nitpick, but we've been recommending folks use `depends_build-append`.

`depends_*-append` is a better practice because you ensure you do not mistakenly overwrite any previous `depends_*` specifications that might have already been set by importing any Portgroups.

Reply:
You're right, fixed.